steps/monitoring-apps-logs/1.0.1/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Mobile apps logs
+summary: |
+  Check if your Android app kept any logs in your release build
+description: "Check if an Android application contain any logs in the release app\nIf
+  yes, this step will fail and generate a failure where you can see the count \n\n###
+  Troubleshooting\n\n- Be aware, you have to launch this step after [apps-decompiler](https://github.com/imranMnts/bitrise-step-apps-decompiler)
+  to have informations from your APK\n"
+website: https://github.com/odemolliens/bitrise-step-monitoring-apps-permissions
```

Comment:
How permissions monitoring is related to logs monitoring?

Reply:
I missed to update these links, I did it in #2977 thanks